valijson.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: valijson
+  version: "1.0.1"
+  epoch: 0
+  description: "C++ library for validating JSON schema"
+  copyright:
+    - license: "BSD-2-Clause"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - cmake
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/tristanpenman/valijson.git
+      tag: v${{package.version}}
+      expected-commit: f7399c1a244982632671906d17f3ea77f3ccfc67
+
+  - uses: cmake/configure
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: valijson-static
+    description: "C++ library for validating JSON - static libraries"
+    pipeline:
+      - uses: split/static
+
+  - name: valijson-dev
+    description: "C++ library for validating JSON - development headers"
+    pipeline:
+      - uses: split/dev
+
+update:
+  enabled: true
+  github:
+    identifier: tristanpenman/valijson
+    strip-prefix: v
```

Comment:
This will require some additional logic, as the release names are in the format: 'Valijson v1.0.1'. i.e need to strip off the preceding 'Valijson '. I don't have an example to hand but this is implemented for other packages so we should be able to find something to reference